README.md

```diff
@@ -55,8 +55,7 @@ The message is a concatenated string  generated from the following parts:
 * `Request-Body`: The raw body of the HTTP request
   * Note that in the message, the body is hashed using the MD5 algorithm
 * `Content-Type`: The lowercase value of the "Content-type" header
-* `Date`: The value of the "Date" header
-  * The implementation may also read the timestamp from a custom header, e.g. `x-acquia-timestamp`
+* `Date`: The value of the "Date" header. The Date general-header field represents the date and time at which the message was originated, having the same semantics as orig-date in RFC 822. The field value is an HTTP-date, as described in section 3.3.1; it MUST be sent in RFC 1123 [8]-date format.
```

Comment:
Normally you don't mess with the Date header and it should not be sent with GET requests, so I think this is problematic